tockloader-cli/src/display.rs

```diff
@@ -0,0 +1,242 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright OXIDOS AUTOMOTIVE 2024.
+
+use tockloader_lib::attributes::{
+    app_attributes::AppAttributes, hardware_attributes::HardwareAttributes,
+};
+
+pub async fn print_list(app_details: &mut [AppAttributes], info: bool) {
```

Comment:
I would split this into two functions:
`print_list_basic(app_detials)` and
`print_list_detailed(app_details)`

to remove the need for the `info` boolean.